README.md

```diff
@@ -0,0 +1,56 @@
+# batwatch
+
+_batwatch_ monitors a system's batteries and UPS units for state changes and sends
+notifications via _gpgmailer_.
+
+Bug fixes are welcome!
```

Comment:
The first part of the copyright line was still relevant. "batwatch is licensed under the GNU GPLv3."